src/ui/MLFacet/MLFacet.ts

```diff
@@ -248,6 +248,28 @@ export class MLFacet extends Component {
     this.handleFacetValuesChanged();
   }
 
+  /**
+   * Requests an additional amount of values equal to the [`number of values`]{@link MLFacet.options.numberOfValues}.
+   *
+   * Does trigger a query automatically.
+   */
+  public showMoreValues(): void {
+    this.ensureDom();
+    this.logger.info('Show more values', this.mLFacetQueryController.increaseNumberOfValuesToRequest());
```

Comment:
Should the call to `increaseNumberOfValuesToRequest` be inside the log call? I think it would give `undefined`? Should we log this action to the console?

Same for line 269.